tasks/main.yml

```diff
@@ -35,7 +35,7 @@
     mode: 0644
 
 - name: Find latest release of .war file
-  shell: curl https://api.github.com/repos/GSA/catalog-fgdc2iso/releases/32097298/assets | jq '.[0].browser_download_url'
+  shell: curl -s https://api.github.com/repos/GSA/catalog-fgdc2iso/releases/32097298/assets | jq -r '.[0].browser_download_url'
```

Comment:
Can you make this a variable? That we we can update it if we need to in datagov-deploy without having to make another code change here.